src/js/azdata/azdataQueryPlan.js

```diff
@@ -71,6 +71,7 @@ azdataQueryPlan.prototype.init = function (container, iconPaths) {
     }
 
     graph.getModel().beginUpdate();
+    console.log(this.queryPlanGraph);
```

Comment:
Could probably go without this console.log statement.

Reply:
Oops, forgot that.